### PR TITLE
LESS version added

### PR DESCRIPTION
**Pull request by t1gr0u**
This is a first draft of the LESS version in "css/less", compile "main.less"

### CODE REVIEW

**Comment by addyosmani** [comment]
Thanks for the PR @t1gr0u!. @panosru what are your thoughts on the above?

**Comment by panosru** [comment]
@addyosmani Well, not that is not a good job, gz @t1gr0u good work on that one and thanks for the PR I believe some people will find it useful, although I'm not among those people not that is not good, in fact it is, I just think that it could be unproductive in the bottom line having jquery and everything else in .less would lead you to maintain every minor change those products will have, eg jquery ui will come with new version and css might have some changes, what I would prefer to do is just grab that new css file and override my old ones and not check the change-log or use a diff tool in order to apply the changes and this applies to all other situations like this one, I give exceptions only in certain things.

In other words, my goal would be to keep vendors easy updatable and not maintain their modifications, spending time on vendors for reasons like this I believe is pointless.

Please @t1gr0u don't get me wrong on that one, it is just my point of view and you did a good job :)

**Comment by dciccale** [comment]
great job @t1gr0u !!

+1

**Comment by dsc8x** [comment]
thank you @t1gr0u,  nice work! (-:

**Comment by addyosmani** [comment]
Cool. We might try pulling in some of this work into the 2.0 branch that's being worked on by @panosru at the moment if it hasn't already been done. 

I should note that how we address the jQuery plugins in Bootstrap 2.0 has yet to be decided (I see those have been included in the latest commits on this thread). In 1.0 the direction taken was to try substituting with jQuery UI components where possible, but with things like the tooltips/popovers a call has to be made about whether we use UI-compatible plugins for that (say, based on the widget factory), or just use the plugins Bootstrap 2.0 come with.

I _don't_ want us to end up in a situation where we're including all of the Bootstrap 2.0 plugins because obviously that will negate the point of us achieving the functionality with jQuery UI where possible.

**Comment by addyosmani** [comment]
The project now has LESS support so closing this issue.
